### PR TITLE
Update download-pipeline-artifact.md

### PR DESCRIPTION
**Pull request by jahsu-MSFT**
Was doing some experiment and noticed that the current examples doesn't seem to be right.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@jahsu-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/tasks/utility/download-pipeline-artifact.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Use this task in a build or release pipeline to download pipeline artifacts from
   inputs:
     buildType: 'specific'
```
Comment (willsmythe) [inline docs/pipelines/tasks/utility/download-pipeline-artifact.md]: buildType should be `source`

File: docs/pipelines/tasks/utility/download-pipeline-artifact.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Use this task in a build or release pipeline to download pipeline artifacts from
...
     project: 'FabrikamFiber'
-    pipelineId: 12
+    pipeline: 12
     buildVersionToDownload: 'latest'
```
Comment (willsmythe) [inline docs/pipelines/tasks/utility/download-pipeline-artifact.md]: should be `runVersion`

File: docs/pipelines/tasks/utility/download-pipeline-artifact.md (hunk 1 above)
Comment (willsmythe) [reply docs/pipelines/tasks/utility/download-pipeline-artifact.md]: (need to change this in a few places)

File: docs/pipelines/tasks/utility/download-pipeline-artifact.md [hunk 3]
```diff
@@ -91,8 +91,9 @@ Use this task in a build or release pipeline to download pipeline artifacts from
     artifact: 'WebApp'
     targetPath: $(Build.SourcesDirectory)/bin
```
Comment (willsmythe) [inline docs/pipelines/tasks/utility/download-pipeline-artifact.md]: should be `path`

File: docs/pipelines/tasks/utility/download-pipeline-artifact.md [hunk 4]
```diff
@@ -75,7 +75,7 @@ Use this task in a build or release pipeline to download pipeline artifacts from
   inputs:
     buildType: 'specific'
     project: 'FabrikamFiber'
-    pipelineId: 12
+    pipeline: 12
     buildVersionToDownload: 'latestFromBranch'
     branchName: 'refs/heads/master'
```
Comment (willsmythe) [inline docs/pipelines/tasks/utility/download-pipeline-artifact.md]: should be `runBranch`

File: docs/pipelines/tasks/utility/download-pipeline-artifact.md [hunk 5]
```diff
@@ -91,8 +91,9 @@ Use this task in a build or release pipeline to download pipeline artifacts from
     artifact: 'WebApp'
     targetPath: $(Build.SourcesDirectory)/bin
     project: 'FabrikamFiber'
-    pipelineId: 12
+    pipeline: 12
     buildVersionToDownload: 'specific'
+    pipelineId: 40
```
Comment (willsmythe) [inline docs/pipelines/tasks/utility/download-pipeline-artifact.md]: should be `runId` ... helps avoid confusion with the `pipeline` (i.e. definition) input

**Comment by ktoliver** [comment]
@jahsu-MSFT - Would you update the pull request per author feedback? Thanks.

**Comment by PRMerger17** [comment]
@jahsu-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@jahsu-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@jahsu-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@jahsu-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by jahsu-MSFT** [comment]
Thanks! Can someone with write access help merge the PR?

**Pull request merged**